src/components/detector.ts

```diff
@@ -110,17 +114,21 @@ export default class Detector extends EventEmitter implements IDetector {
    * @memberof Detector
    */
   async disable(idleTimeMs: number = 5000): Promise<void> {
-    let isEnabled = await this.isEnabled();
-    if (!isEnabled) return; // already disabled
```

Comment:
Maybe we should keep the two lines above. No need to disable if already disabled!

Reply:
I think you should right now. Because the camera starts receiving requests before the detector has finished initializing, the camera will first report autozoom is off while the detector is initializing (takes a few additional seconds). So if you run disable right after boot it would then not disable. I think this is safer.